packages/oui-tile/_mixins.less

```diff
@@ -0,0 +1,148 @@
+@import './_variables';
+
+#oui {
+  .tile-base(
+    @tile-selector,
+    @dropdown-menu-selector,
+    @icon-selector
+  ) {
+    padding: @oui-tile-padding;
+    background-color: @oui-tile-background-color;
+    border: 1px solid @oui-tile-border-color;
+    border-radius: @oui-tile-border-radius;
+    box-shadow: @oui-tile-box-shadow;
+
+    &__title {
+      margin-left: @oui-tile-item-side-spacing;
+      margin-right: @oui-tile-item-side-spacing;
+      font-weight: 600;
+      color: @oui-tile-title-color;
+    }
+
+    &__button {
+      margin: 0;
+      padding-left: @oui-tile-item-side-spacing;
+      padding-right: @oui-tile-item-side-spacing;
+      font-weight: 600;
+      text-align: left;
+      text-decoration: none;
+
+      &:disabled {
+        text-decoration: none;
+      }
+
+      .@{icon-selector} {
+        color: @oui-tile-button-color;
+
+        &::before {
+          font-size: rem-calc(12);
+        }
+      }
+    }
+
+    &__item {
+      position: relative;
+
+      &::before {
```

Comment:
Because it's not a simple border-top (Cf. `margin: 0 @oui-tile-item-side-spacing;`).
However, it's a draft, this will probably change in few days...